packages/core/features/src/connect.ts

```diff
@@ -1,5 +1,8 @@
 import type { WalletAccount } from '@wallet-standard/base';
 
+/** Name of the feature. */
+export const Connect = 'standard:connect';
```

Comment:
This could be `Connect`, `ConnectName`, or `StandardConnect`. It might be generally better to prefix all the types here with `Standard` and deprecate the current ones, inline with how we prefix all the chain-specific features e.g. `solana:` with `Solana`.

Reply:
Agreed. Do you want to do this here for the newly introduced constants, or in a follow-up PR that also renames the types?

Reply:
I'll merge and do it as a quick followup before actually using these